README.md

```diff
@@ -26,6 +26,9 @@ conda install -c conda-forge numpy scipy matplotlib netCDF4 xarray dask bottlene
 ```
 
 ## Running the analysis
-  1. create a configuration file by copying and modifying `config.template` or one of the example files in `configs`
-  2. run: `./run_analysis.py config.myrun`, where `config.myrun` is the config file you created
-  3. If you want to run a subset of the analysis, you can either modify the `generate` option under `[output]` in the config file or use the `--generate` flag on the command line.  See `config.template` for more details.
+  1. create a configuration file (say `config.myrun`) by copying and modifying 
+     `config.default` or one of the example files in `configs`.  Note, you only
+     need include entries that are modified from `config.default` in your 
+     confguration file; entries from `config.default` will be used by default.
```

Comment:
small typo (configuration).
Also, a comment: does this mean that, if I do *not* want to modify config.defaults at all, I do not need to do anything (just get the config specific files from configs)?
Instead, if I do want to modify config.default some, how do you suggest I would proceed: 
1) copy configs/$machine/template/config file to config.myrun
2) modify config.myrun to include the changes I want from config.default?